### ISSUE TITLE
JTableCellFixture will always initialize 'cell' to null

### ISSUE THREAD

**Issue by martinrleon** [alexruiz/fest-swing-1.x#12]
Constructor uses 'cel()' instead of 'cell', which will always set cell to null because cell() returns cell which isn't initialized :)

https://github.com/alexruiz/fest-swing-1.x/blob/master/fest-swing/src/main/java/org/fest/swing/fixture/JTableCellFixture.java#L69

**Comment by alexruiz** [comment]
Fixed! Thanks Martin!

https://github.com/alexruiz/fest-swing-1.x/commit/20a9351afb37667d4bec0fbb600f26defbe73dea

**Comment by martinrleon** [comment]
FYI, I got my test suite running again under Java 7 on OS X with local fixes to the issues I submitted.  In other words, I probably won't be submitting any more issues.  But I will be anxiously waiting for the release with all the fixes.  :)

Thank you and Yvonne for all your hard work on this project, this tool is indispensable for us at this point!

Best regards,

Martin
http://www.linkedin.com/in/martinrleon/
